code/datums/abilities/ability_parent.dm

```diff
@@ -223,6 +223,12 @@
 
 		points -= cost
 
+	proc/addPoints(add_points)
```

Comment:
```suggestion
	proc/addPoints(add_points, target_ah_type)
```
Imo it's good practice to have the same arguments on all overrides of a proc. But opinions might differ.